src/Components/AddPresentationComponent.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import TextField from "@material-ui/core/TextField";
+import DialogActions from "@material-ui/core/DialogActions";
+import Grid from "@material-ui/core/Grid";
+import axios from "axios";
+
+export default function AddPresentationComponent() {
+    const [open, setOpen] = React.useState(false);
+
+    //presentation details
+    const [company, setCompany] = React.useState("");
+    const [address, setAddress] = React.useState("");
+    const [date, setDate] = React.useState(null);
+    const [attending, setAttending] = React.useState(null);
+    const [description, setDescription] = React.useState("");
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+
+    const AddPresentation = async () => {
+        try {
+            const response = await axios.post('http://localhost:8080/addPresentation', {
+                company : company,
+                address : address,
+                date : date,
+                attending : attending,
+                description : description
+            });
+        }
+        catch (e) {
+            console.error(e);
+        }
```

Comment:
Det går og an å gjør sånn i stedet for try/catch:
`fetch('https://no-such-server.blabla') // rejects
  .then(response => response.json())
  .catch(err => consoloe.log(err))`
https://javascript.info/promise-error-handling